src/main/java/wellnus/reflection/ExitCommand.java

```diff
@@ -4,8 +4,11 @@
 import wellnus.exception.BadCommandException;
 
 import java.util.HashMap;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class ExitCommand extends Command {
+    private static final Logger LOGGER = Logger.getLogger(ReflectionManager.class.getName());
```

Comment:
I would highly recommend avoiding the usage of Java reflection for something so basic, since the intention here is just to get something that's compile-time constant. Perhaps an attribute exposed by `ReflectionManager` would be more appropriate and achieve a similar goal. Java reflection is best avoided unless you understand what you're doing I feel